examples/python/image_storage.py

```diff
@@ -27,23 +27,25 @@
 from urllib import request
 import os
 
-# set up the spark session
-spark = SparkSession.builder \
-    .appName("image_storage") \
-    .master("local[*]") \
-    .config("spark.sql.extensions", "io.delta.sql.DeltaSparkSessionExtension") \
-    .config("spark.sql.catalog.spark_catalog", "org.apache.spark.sql.delta.catalog.DeltaCatalog") \
-    .config("spark.sql.sources.parallelPartitionDiscovery.parallelism", "4") \
-    .getOrCreate()
+# # If running as part of the integration tests, set up the spark session
+# spark = SparkSession.builder \
+#     .appName("image_storage") \
+#     .master("local[*]") \
+#     .config("spark.sql.extensions", "io.delta.sql.DeltaSparkSessionExtension") \
+#     .config("spark.sql.catalog.spark_catalog", "org.apache.spark.sql.delta.catalog.DeltaCatalog") \
+#     .config("spark.sql.sources.parallelPartitionDiscovery.parallelism", "4") \
+#     .getOrCreate()
```

Comment:
I think we can remove this entirely. The integration tests run with the current code below (I think, can you verify?), and by nature we need the integration tests to run automatically without code changes.

Reply:
I removed the commented out spark session and verified it does run as a test.